CHANGELOG.md

```diff
@@ -671,8 +672,8 @@ Released 2020-06-16
 
 ## 🛑 Breaking changes 🛑
 
-- `isEnabled` configuration option removed (#909) 
-- `thrift_tchannel` protocol moved from `jaeger` receiver to `jaeger_legacy` in contrib (#636) 
+- `isEnabled` configuration option removed (#909)
+- `thrift_tchannel` protocol moved from `jaeger` receiver to `jaeger_legacy` in contrib (#636)
 
```

Comment:
If you want to fix unnecessary whitespaces I am fine, but please do it in a separate PR.

Reply:
ack